components/size_aware_image.jsx

```diff
@@ -230,6 +356,43 @@ export default class SizeAwareImage extends React.PureComponent {
         );
     }
 
+    isInternalImage = () => {
+        return this.props.fileInfo !== undefined;
+    }
```

Comment:
this can just be a boolean

```suggestion
    isInternalImage = this.props.fileInfo !== undefined;
```